docs/sphinx/source/developer_notes.rst

```diff
@@ -185,8 +185,11 @@ If you get an error like ``Pandoc wasn't found``, you can install it with conda:
 The built documentation should be in ``rdtools/docs/sphinx/build`` and opening
 ``index.html`` with a web browser will display it.
 
-Code requirements
------------------
+Contributing
+------------
+
+Community participation is welcome!  New contributions should be based on the
+``development`` branch as the ``master`` branch is used only for releases.  
```

Comment:
Should we take this out?

Reply:
I thought our discussion with @cdeline ended with "let's keep the development branch workflow, but be better about smaller more frequent releases"

Reply:
If I have that right, we can leave it for now

Reply:
Yeah, I'm fine with keeping this.  Doing merges from development during releases is handy.